### ISSUE TITLE
[osf] project search seems to only search through expanded collections

### ISSUE THREAD

**Issue by csoderberg** [CenterForOpenScience/osf.io#3086]
Steps to reproduce:
* Go to your project dashboard
* With 'all my projects' collapsed, search for a project title that you have.

Expected behavior:
* The search should find the project you are looking for

Actual behavior:
* Nothing comes up. The search will only return a project if the 'all my projects' tab is expanded. So, it seems to just be searching through whatever is currently visible on the project dashboard, not anything underneath the visible level. 

This occurs on Safari 10.9.5 and Firefox 38.0.5, but does NOT occur in Chrome. I don't have IE so I couldn't check that.

**Comment by brianjgeiger** [comment]
Duplicate of https://github.com/CenterForOpenScience/osf.io/issues/2779 (though you wouldn't know it without reading through the discussion)

**Comment by brianjgeiger** [comment]
Alllthough, it looks like they may be focusing on files tab for the fix, so maybe we'll keep this open.

**Comment by csoderberg** [comment]
Yeah, I saw that and wasn't sure if this was different enough to get it's
own issue or not? But it seemed to be talking about different parts of th
system, so I just made one and hoped for the best.

On Thu, Jun 11, 2015 at 4:33 PM, Brian J. Geiger <notifications@github.com>
wrote:

> Alllthough, it looks like they may be focusing on files tab for the fix,
> so maybe we'll keep this open.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/issues/3086#issuecomment-111270970>
> .
>



-- 
Courtney Soderberg, Ph.D
Statistical and Methodological Consultant
Center for Open Science

**Comment by XTech2K** [comment]
I just noticed that there is also an odd connection with #3235; if you add something to a collection, it will always appear in the search until the collection is deleted, even if it is removed from the collection.
